config/locales/administrate.fr.yml

```diff
@@ -6,6 +6,8 @@ fr:
       destroy: Supprimer
       edit: Modifier
       show: Détails
+      new: Nouveau
```

Comment:
This is tricky. In French there is the notion of feminine / masculine for objects. Nouveau only applies to masculine, making the button wrong. I would go for "Création" instead.